ledger/sandbox/src/main/scala/com/digitalasset/platform/indexer/RecoveringIndexer.scala

```diff
@@ -22,14 +22,15 @@ import scala.util.{Failure, Success}
   * @param restartDelay Time to wait before restarting the indexer after a failure
   * @param asyncTolerance Time to wait for asynchronous operations to complete
   */
-class RecoveringIndexer(
+final class RecoveringIndexer(
     scheduler: Scheduler,
     restartDelay: FiniteDuration,
     asyncTolerance: FiniteDuration,
-    loggerFactory: NamedLoggerFactory,
-) {
+)(implicit logCtx: LoggingContext) {
   private implicit val executionContext: ExecutionContext = DirectExecutionContext
-  private val logger = loggerFactory.getLogger(this.getClass)
+  private val logger = ContextualizedLogger.get[this.type]
+
+  val lastHandle = new AtomicReference[Option[Resource[IndexFeedHandle]]](None)
```

Comment:
I think this line snuck in during a rebase. It's been deleted; it shouldn't be necessary.

Reply:
Definitely, good catch, thanks!

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/03d47341bd98899f3737afa61bb6d3404d3456f4